tests/test_jx/test_metadata.py

```diff
@@ -112,7 +112,7 @@ def test_get_nested_columns(self):
                         {"b": "y", "v": 3}
                     ]},
                     {"o": 2, "_a": {"b": "x", "v": 5}},
-                    {"o": 3, "_a": [{"b": "x", "v": 7}]},
+                    {"o": 3, "_a": {"b": "x", "v": 7}},
```

Comment:
using object here,  as in snowflake list with one object should be represented by that one object only as you mentioned before.